CHANGELOG.md

```diff
@@ -1,37 +1,20 @@
-### Added
-
-- WSO2 Open Banking Business Intelligence support for Australian Specification (refer to [issue](https://github.com/wso2/docker-open-banking/issues/111))
-
-## [v2.0.0.2] - 2021-01-13
-
-### Fixed
-
-- Fix product name change of Open Banking Identity & Access Management Module in OB 2.0 release (refer to [issue](https://github.com/wso2/docker-open-banking/issues/103))
-
-## [v2.0.0.1] - 2021-01-06
-
 ### Added
 
-- Open Banking UK and BERLIN specifications support	- Alpine and Ubuntu based Docker resources for WSO2 Open Banking API Manager `2.0.0`, WSO2 Open Banking Identity Server
-  as Identity & Access Management Module version `2.0.0` and WSO2 Open Banking Business Intelligence Dashboard and Worker profiles `2.0.0`
-- Upgrade profile based dockerfiles to use Adopt OpenJDK `jdk8u272-b10` as the base image
-- Docker Compose resources for OBAM, OBIAM with OBBI deployment patterns based on UK and Berlin specifications
-- Docker Compose resources for OBAM with OBIAM deployment patterns based on UK and AU specifications  
-- Solution is based on WSO2 APIM 3.1.0 and WSO2 ISKM 5.10.0  
-- Previous XML based configurations are carried out using TOML based configurations. Please refer the documentation for detailed mapping of configuration changes [Map Configurations](https://docs.wso2.com/display/OB200/Map+XML+Configurations+to+TOML+Configurations)
-- For detailed information of the WSO2 Open Banking 2.0.0 product release, please see the GitHub release notes [Releases](https://github.com/wso2-enterprise/financial-open-banking/releases)
+- Alpine and Ubuntu based Docker resources for WSO2 Open Banking API Manager Accelerator Module version `3.0.0` and WSO2 Open Banking Identity & Access Management Accelerator Module version `3.0.0`
+- Docker Compose resources for OBAM with OBIAM Accelerator
+- Solution is based on WSO2 APIM 4.0.0 and WSO2 IS 5.11.0  
+- Upgrade profile based dockerfiles to use docker.wso2.com/wso2is:5.11.0.0 and docker.wso2.com/wso2is:5.11.0.0 as the base images for OBAM and OBIAM respectively.
+- Previous XML based configurations are carried out using TOML based configurations. Please refer the documentations for detailed mapping of configuration changes 
+  - [Configuring Identity Server](https://ob.docs.wso2.com/en/latest/install-and-setup/configuring-identity-server-for-ob/)
+  - [Configuring API Manager](https://ob.docs.wso2.com/en/latest/install-and-setup/configuring-api-manager-for-ob/)
+- For detailed information of the WSO2 Open Banking 3.0.0 product release, please see the GitHub release notes [Releases](https://github.com/wso2-enterprise/financial-open-banking/releases)
 
-For detailed information on the tasks carried out during this release, please see the GitHub milestone [v2.0.0.1](https://github.com/wso2/docker-open-banking/milestone/11)
+For detailed information on the tasks carried out during this release, please see the GitHub milestone [v3.0.0.1](https://github.com/wso2/docker-open-banking/milestone/11)
```

Comment:
@AmilaSamith if you haven't created a GitHub Milestone with the issues covered in this release, let's remove this line. Currently, it points to an older milestone.

Reply:
Removed line related to milestone